release.md

```diff
@@ -1,41 +1,190 @@
 # Versioning and Releases
 
-Cirq is currently (as of July 11, 2018) alpha, and so has a MAJOR version 
-of 0. Below is info on how we version releases, and how the releases 
-themselves are created. Note that development is done on the `master` 
-branch, so if you want to use a more stable version you should use one 
-of the [releases](https://github.com/quantumlib/Cirq/releases) or 
-install from pypi.
+Cirq is currently (as of May 2020) alpha, and so has a MAJOR version
+of 0. Below is info on how we version releases, and how the releases
+themselves are created. Note that development is done on the `master`
+branch, so if you want to use a more stable version you should use one
+of the [releases](https://github.com/quantumlib/Cirq/releases) or
+install from pypi using `pip install cirq`.  Nightly releases can be
```

Comment:
Nit: Nightly isn't quite correct word since it is per merge to master.  "Release from the latest commit to master"

Reply:
Whoa, I didn't realize that.  Has this changed over the last few months?